tests/test_dataset_registry.py

```diff
@@ -72,6 +72,8 @@ def test_dataset(dataset_name: str, dummy_dataloader_hparams: DataLoaderHparams)
     hparams_cls = dataset_registry[dataset_name]
     if issubclass(hparams_cls, WebDatasetHparams):
         return pytest.skip("Skipping test for webdatasets")
+    elif issubclass(hparams_cls, StreamingDatasetHparams):
```

Comment:
Curious why is this being skipped?

Reply:
My bad, didn't mean to commit this file. Hparams tests will come with the new examples of `StreamingDataset` in Part 2...